chainer/training/extensions/__init__.py

```diff
@@ -17,3 +17,5 @@
 from chainer.training.extensions.value_observation import observe_lr  # NOQA
 from chainer.training.extensions.value_observation import observe_value  # NOQA
 from chainer.training.extensions.variable_statistics_plot import VariableStatisticsPlot  # NOQA
+from chainer.training.extensions.warm_up import WarmUp  # NOQA
```

Comment:
Could you make a separate PR for these extensions, as these are not related to iDeep?

Reply:
ok, I will make a separate PR for these extensions. Thanks.